RFC-008-release-versioning.md

```diff
@@ -0,0 +1,267 @@
+# RFC008 - Release Versioning Strategy
+
+## Proposed by
+
+Aaron Peters
+
+
+## Current status
+
+See the status indicated on the pull request description.
+
+
+## Proposal overview
+
+As The Good Docs Project moves towards a formal release structure, this RFC proposes a standard versioning scheme to apply at least to the project's Product Outputs, as defined by outputs directly employed by users in their own documentation efforts. For example, this includes the collection of templates (which users copy as a starting point for their own documents).
+
+TGDP will use the Semantic Versioning convention for the versioning of individual Product Outputs (as at November 2021, this includes the template collection and the DocTools website package).
+
+> :triangular_flag_on_post: For the purposes of this RFC, "features" and "functionality" can refer to new **content**, such as additional sections or boilerplate text, added to doc templates.

```

Comment:
Non-blocking: Is it worth us using the same Product/Non framework from RFC006 here? Are we pseudo-duplicating attempts that delineation here?